tox.ini

```diff
@@ -43,7 +43,7 @@ basepython = python2.7
 deps=
     -r{toxinidir}/docs/requirements.txt
 commands=
-    rm -r {toxinidir}/docs/_build
+    rm -rf {toxinidir}/docs/_build
```

Comment:
So it turns out this works as long as you’ve ever done a docs build before.

If you checkout the codebase and run this command, it errors trying to clean up a directory that it hasn’t been able to create yet.